.github/workflows/build-image.yml

```
@@ -28,21 +28,25 @@ jobs:
         - name: cuda12.4
           dockerfile: cuda12.4
           tags: superbench/main:cuda12.4
+          platforms: linux/amd64, linux/arm64
```

```suggestion
          platforms: linux/amd64
```
let's skip the arm64 build in CI first, the emulation seems too slow (10x than amd64)..